### ISSUE TITLE
Support instant installation of standard/common ontologies

### ISSUE THREAD

**Issue by yoavnash** [simphony/osp-core#455]
It would be nice to be able to install standard ontologies like so:
``` sh
pico install foaf
pico install dcat
```

**Comment by urbanmatthias** [comment]
Hi @eich-2020,

this might be a good second issue.

Use the information provided [here](https://simphony.readthedocs.io/en/dev/owl.html) to create a yml file for [foaf](http://xmlns.com/foaf/spec/) and [dcat](https://www.w3.org/TR/vocab-dcat-2/).

You can download them and take a look with protege: http://xmlns.com/foaf/spec/index.rdf / https://www.w3.org/ns/dcat.ttl

- The identifier should be `foaf`/`dcat`
- The `ontology_file` should be the linked rdf or turtle file: http://xmlns.com/foaf/spec/index.rdf / https://www.w3.org/ns/dcat.ttl
- `reference_by_label` should be False.
- namespaces should be only `foaf="http://xmlns.com/foaf/0.1/"` / `dct: "http://purl.org/dc/terms/"` as one could see from the prefix sections.
- In osp-core we have the concept that CUDS objects are containers. If we add object B to object A (i.e. `A.add(B, rel=my_rel)`) we add B to the container of A, if `my_rel` is a sub-property of an active property. In EMMO these are e.g. `hasPart`, `encloses`, `hasItem`. Examples for non-active relationships are `overlaps`, `overcrosses`. So here you would have to go through all ObjectProperties in these two ontologies and think about whether it makes sense to make them active or not.
- The default relationship is the one used when the use doesn't specify one when calling `add()`.

**Comment by yoavnash** [comment]
Perhaps a natural future feature that is good to keep in mind is the support of versions. For example:
`pico install emmo@1.0.1`

**Comment by urbanmatthias** [comment]
True, that is important and we'll have to figure out how to support different versions.
For the sake of @eich-2020 's second issue it is enough to only support the latest version.

**Comment by urbanmatthias** [comment]
I created an issue targeting the support of different ontology versions:

**Comment by kysrpex** [comment]
Suggestion from @pablo-de-andres: let the user also easily install ontologies from other scientific domains, e.g. Bioinformatics.

**Comment by kysrpex** [comment]
This [link](https://lov.linkeddata.es/dataset/lov/) might help.

**Comment by yoavnash** [comment]
It seems to already work for FOAF, and for DCAT there are technical issues that are being addressed separately (e.g, #594).
What is then left to do here?

**Comment by kysrpex** [comment]
> It seems to already work for FOAF, and for DCAT there are technical issues that are being addressed separately (e.g, #594).
> What is then left to do here?

To recap what I told you privately, essentially we would need more YML files included with OSP-core (we have them alrady for FOAF and DCAT), as they include the necessary metadata to download the ontologies. However, it might be worth to leave this on hold until we solve the problem of ontology version management.
